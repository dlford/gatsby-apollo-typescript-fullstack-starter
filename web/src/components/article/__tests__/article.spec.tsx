import React from 'react'
import { render } from '@testing-library/react'
import Article from '../'

describe('Article', () => {
  it('renders correctly', () => {
    const { asFragment } = render(
      <Article>
        <h1>Test</h1>
      </Article>,
    )
    expect(asFragment()).toMatchSnapshot()
  })
})
