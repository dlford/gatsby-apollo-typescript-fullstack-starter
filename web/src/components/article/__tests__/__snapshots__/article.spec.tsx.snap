// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Article renders correctly 1`] = `
<DocumentFragment>
  <article
    class="css-12xddvo"
  >
    <h1>
      Test
    </h1>
  </article>
</DocumentFragment>
`;
